command/acl/token/update/token_update.go

```diff
@@ -184,15 +199,31 @@ func (c *cmd) Run(args []string) int {
 			}
 		}
 	} else {
-		t.Policies = nil
 
-		for _, policyName := range c.policyNames {
+		hasAddPolicyFields := len(c.addPolicyNames) > 0 || len(c.addPolicyIDs) > 0
+		hasPolicyFields := len(c.policyIDs) > 0 || len(c.policyNames) > 0
+
+		if hasPolicyFields && hasAddPolicyFields {
+			c.UI.Error("Cannot specify both add-policy-id/add-policy-name and policy-id/policy-name")
```

Comment:
This could use some detail to explain to the user how to use the flags correctly
```suggestion
			c.UI.Error("Cannot combine the use of policy-id/policy-name flags with add- variants. To set or overwrite existing policies, use -policy-id or -policy-name. To append to existing policies, use -add-policy-id or -add-policy-name.")
```